src/get-started/create-service.md

```diff
@@ -376,20 +364,19 @@ impl Transaction for TxTransfer {
 
 ## Implement API
 
-Next, we need to implement the node API with the help of [Iron framework][iron].
-With this aim we declare a struct which implements the `Api` trait.
-The struct will contain a channel, i.e. a connection to the blockchain node
+Next, we need to implement the node API with the help of [Actix-web framework][actix-web].
+With this aim we declare a blank struct with set of methods with the following signature:
```

Comment:
Many 'with' in this sentence. Perhaps we could rewrite it to something like:
'With this aim we declare a blank struct that includes a set of methods with the following signature'

Or if the meaning of 'declare a blank struct with set of methods' means using these methods, then:
'With this aim we declare a blank struct using a set of methods with the following signature'

Reply:
First variant is preferred for me.